src/app/AttributeAccessInterface.h

```diff
@@ -109,7 +117,7 @@ class AttributeValueEncoder
             VerifyOrReturnError(!mAttributeValueEncoder.mIsFabricFiltered ||
```

Comment:
While we are here, we should `VerifyOrReturnError(aArg.GetFabricIndex() != kUndefinedFabricIndex);`, right?